composer.json

```diff
@@ -17,7 +17,7 @@
         "behat/mink-extension": "^2.2",
         "behat/mink-browserkit-driver": "^1.3",
         "behat/symfony2-extension": "^2.1",
-        "symfony/swiftmailer-bundle": "^2.5"
+        "symfony/swiftmailer-bundle": "^2.5|^3.2"
```

Comment:
is an OR (in the docs it says or is ||)? if yes, why use it instead of requiring an specific one?